Source/JavaScriptCore/runtime/TemporalPlainDate.cpp

```diff
@@ -266,9 +270,60 @@ std::array<std::optional<double>, numberOfTemporalPlainDateUnits> TemporalPlainD
             throwRangeError(globalObject, scope, "year property must be finite"_s);
             return { };
         }
+        any = TemporalAnyProperties::Some;
+    }
+
+    if (monthCodePresent) {
+        if (!otherMonth) {
+            throwRangeError(globalObject, scope, "Invalid monthCode property"_s);
+            return { };
+        }
+        if (!month)
+            month = otherMonth->monthNumber;
+        else if (month.value() != otherMonth->monthNumber) {
+            throwRangeError(globalObject, scope, "month and monthCode properties must match if both are provided"_s);
+            return { };
+        }
+    }
+
+    TemporalOverflow overflow = toTemporalOverflow(globalObject, optionsValue);
+
+    RETURN_IF_EXCEPTION(scope, { });
+
+    // Duplicate code from TemporalPlainDate::toPlainDate so we can convert from
+    // double to int32_t / unsigned here
+    if (year && !ISO8601::isYearWithinLimits(*year)) {
+        throwRangeError(globalObject, scope, "year is out of range"_s);
+        return { };
+    }
+
+    int32_t yearToUse = defaultYear;
+    if (year)
+        yearToUse = static_cast<int32_t>(*year);
+    unsigned monthToUse = defaultMonth;
+    if (month) {
+        if (overflow == TemporalOverflow::Constrain)
+            monthToUse = std::min<unsigned>(*month, 12);
+        else {
+            if (!(month >= 1 && month <= 12)) {
```

Comment:
`[[unlikely]]`

Reply:
Fixed in [348c785](https://github.com/WebKit/WebKit/pull/52251/commits/348c785580dc01981e23b15ddbfdafabda503920)